www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -17,6 +17,22 @@
   @include breakpoint("large-") {
     margin-bottom: 20px;
   }
+
+  h3 {
```

Comment:
Some enhanced styling for the `marketing-col`. Are we avoiding tabs at this stage?